### PR TITLE
Spell Checking / Updated Style Checking

### PR DESCRIPTION
**Pull request by mDuo13**
Adds configuration to use updates to the experimental Dactyl Style checker (including a spelling checker) and fix numerous cases of unrecognized words. Most of the words are just names, or code that wasn't marked as such, but it did catch a few typos:

-    indentifying
-    apppropriate
-    asychronous
-    paramters
-    psuedo-transaction
-    ECSDA
-    addresss

Most of the changes are to provide more consistency in naming and code-font usage. In some cases I reworded sentences to avoid using obscure words the spell checker didn't recognize.

There are also a few changes to make the style checker a little happier (like removing instances of "simply") and some (also experimental) settings for grading the readability of the docs.

### CODE REVIEW

**Comment by ryangyoung** [comment]
From what I can tell, LGTM. I ran the style checker on the `spelling` branch and it didn't return any spelling issues:

```
Found 354 issues:
Page: XRP Ledger Overview
   Discouraged word: magnitude (1 instances); suggest 'size' instead.
   Discouraged word: will (1 instances); suggest '(avoid future tense)' instead.
Page: Technical FAQ
   Discouraged word: therefore (1 instances); suggest 'so' instead.
   Discouraged word: will (7 instances); suggest '(avoid future tense)' instead.
   Discouraged word: regarding (1 instances); suggest 'about, of, on' instead.
Page: Cryptographic Keys
   Discouraged phrase: is authorized to (2 instances); suggest 'may' instead.
   Discouraged phrase: in the event of (2 instances); suggest 'if' instead.
   Discouraged word: easily (1 instances); suggest '(omit)' instead.
   Discouraged phrase: similar to (1 instances); suggest 'like' instead.
Page: Deposit Authorization
   Discouraged phrase: comply with (1 instances); suggest 'follow' instead.
   Discouraged word: will (1 instances); suggest '(avoid future tense)' instead.
   Discouraged word: remainder (1 instances); suggest 'rest' instead.
   Discouraged word: accordingly (1 instances); suggest 'so' instead.
   Discouraged phrase: is authorized to (1 instances); suggest 'may' instead.
Page: Transaction Basics
   Discouraged phrase: in the process of (1 instances); suggest '(omit)' instead.
Page: Checks
   Discouraged phrase: similar to (1 instances); suggest 'like' instead.
   Discouraged word: portion (1 instances); suggest 'part' instead.
   Discouraged word: disclose (1 instances); suggest 'show' instead.
   Discouraged phrase: comply with (1 instances); suggest 'follow' instead.
   Discouraged word: will (1 instances); suggest '(avoid future tense)' instead.
Page: Payment Channels
   Discouraged phrase: time period (1 instances); suggest '(either one)' instead.
Page: Issued Currencies Overview
   Discouraged word: ious (1 instances); suggest 'issuances' instead.
   Discouraged phrase: comply with (1 instances); suggest 'follow' instead.
Page: Freezing Issued Currencies
   Discouraged phrase: in order to (1 instances); suggest 'to' instead.
   Discouraged word: ious (1 instances); suggest 'issuances' instead.
Page: Demurrage
   Discouraged word: reflect (1 instances); suggest 'say, show' instead.
   Discouraged phrase: similar to (1 instances); suggest 'like' instead.
   Discouraged word: demonstrate (1 instances); suggest 'prove, show' instead.
Page: Consensus
   Discouraged word: accordingly (1 instances); suggest 'so' instead.
Page: Consensus Principles and Rules
   Discouraged phrase: in addition (1 instances); suggest 'also, besides, too' instead.
Page: Consensus Protections Against Attacks and Failure Modes
   Discouraged word: timely (1 instances); suggest 'prompt' instead.
   Discouraged phrase: in a timely manner (1 instances); suggest 'on time, promptly' instead.
   Discouraged phrase: a number of (1 instances); suggest 'some' instead.
   Discouraged word: disclose (1 instances); suggest 'show' instead.
   Discouraged word: accomplish (2 instances); suggest 'carry out, do' instead.
   Discouraged phrase: similar to (1 instances); suggest 'like' instead.
Page: About Canceling a Transaction
   Discouraged phrase: a number of (1 instances); suggest 'some' instead.
Page: Transaction Malleability
   Discouraged word: component (1 instances); suggest 'part' instead.
   Discouraged word: render (1 instances); suggest 'give, make' instead.
   Discouraged word: designate (2 instances); suggest 'appoint, choose, name' instead.
   Discouraged phrase: similar to (1 instances); suggest 'like' instead.
Page: Known Amendments
   Discouraged phrase: comply with (1 instances); suggest 'follow' instead.
   Discouraged phrase: is authorized to (1 instances); suggest 'may' instead.
   Discouraged word: therefore (1 instances); suggest 'so' instead.
   Discouraged word: encounter (2 instances); suggest 'meet' instead.
   Discouraged word: identical (1 instances); suggest 'same' instead.
Page: Ledger History
   Discouraged word: sufficient (1 instances); suggest 'enough' instead.
   Discouraged word: portion (1 instances); suggest 'part' instead.
Page: Online Deletion
   Discouraged word: feasible (1 instances); suggest 'can be done, workable' instead.
   Discouraged phrase: a number of (2 instances); suggest 'some' instead.
Page: History Sharding
   Discouraged word: witnessed (1 instances); suggest 'saw' instead.
   Discouraged word: sufficient (1 instances); suggest 'enough' instead.
Page: Peer Protocol
   Discouraged word: establish (1 instances); suggest 'set up, create, prove' instead.
   Discouraged word: prioritize (1 instances); suggest 'rank' instead.
   Discouraged phrase: similar to (1 instances); suggest 'like' instead.
   Discouraged word: eliminate (1 instances); suggest 'cut, drop, end' instead.
Page: Transaction Censorship Detection
   Discouraged word: impacted (1 instances); suggest 'affected, changed' instead.
Page: Set Up Secure Signing
   Discouraged word: transmit (1 instances); suggest 'send' instead.
Page: Look Up Transaction Results
   Discouraged word: sufficient (1 instances); suggest 'enough' instead.
   Discouraged word: reflect (1 instances); suggest 'say, show' instead.
   Discouraged word: therefore (1 instances); suggest 'so' instead.
   Discouraged phrase: in addition (1 instances); suggest 'also, besides, too' instead.
Page: Monitor Incoming Payments with WebSocket
   Discouraged word: establish (1 instances); suggest 'set up, create, prove' instead.
   Discouraged phrase: in addition (1 instances); suggest 'also, besides, too' instead.
   Discouraged word: demonstrate (1 instances); suggest 'prove, show' instead.
Page: Send XRP
   Discouraged word: simple (1 instances); suggest '(omit)' instead.
   Discouraged word: viable (2 instances); suggest 'practical, workable' instead.
   Discouraged word: consult (1 instances); suggest 'see' instead.
Page: Reliable Transaction Submission
   Discouraged word: accomplish (1 instances); suggest 'carry out, do' instead.
Page: Assign a Regular Key Pair
   Discouraged word: subsequent (1 instances); suggest 'later, next' instead.
   Discouraged word: transmit (2 instances); suggest 'send' instead.
Page: Change or Remove a Regular Key Pair
   Discouraged word: whereas (1 instances); suggest 'because, since' instead.
   Discouraged word: transmit (1 instances); suggest 'send' instead.
Page: Disable Master Key Pair
   Discouraged word: transmit (1 instances); suggest 'send' instead.
Page: Offline Account Setup Tutorial
   Discouraged word: accurate (2 instances); suggest 'correct, exact, right' instead.
   Discouraged word: simple (2 instances); suggest '(omit)' instead.
   Discouraged word: will (2 instances); suggest '(avoid future tense)' instead.
   Discouraged phrase: in order to (1 instances); suggest 'to' instead.
   Discouraged word: sufficient (1 instances); suggest 'enough' instead.
   Discouraged word: viable (1 instances); suggest 'practical, workable' instead.
Page: Use Checks
   Discouraged phrase: similar to (1 instances); suggest 'like' instead.
Page: Cash a Check for a Flexible Amount
   Discouraged word: accordingly (1 instances); suggest 'so' instead.
Page: Cancel a Check
   Discouraged word: demonstrate (1 instances); suggest 'prove, show' instead.
Page: List XRP as an Exchange
   Discouraged word: easy (1 instances); suggest '(omit)' instead.
   Discouraged phrase: warm wallet (2 instances); suggest 'standby address' instead.
   Discouraged word: wallet (21 instances); suggest 'address, client application, account' instead.
   Discouraged phrase: cold wallet (4 instances); suggest 'issuing address' instead.
   Discouraged phrase: hot wallet (2 instances); suggest 'operational address' instead.
Page: List Your Exchange on XRP Charts
   Discouraged word: will (1 instances); suggest '(avoid future tense)' instead.
Page: Become an XRP Ledger Gateway
   Discouraged word: feasible (1 instances); suggest 'can be done, workable' instead.
   Discouraged word: wallet (1 instances); suggest 'address, client application, account' instead.
Page: System Requirements
   Discouraged word: accurate (1 instances); suggest 'correct, exact, right' instead.
Page: Capacity Planning
   Discouraged phrase: in the near future (1 instances); suggest 'shortly, soon' instead.
   Discouraged word: utilization (2 instances); suggest 'use' instead.
   Discouraged word: encounter (1 instances); suggest 'meet' instead.
   Discouraged word: therefore (2 instances); suggest 'so' instead.
   Discouraged phrase: similar to (1 instances); suggest 'like' instead.
   Discouraged word: reside (1 instances); suggest 'live' instead.
   Discouraged phrase: in the event of (1 instances); suggest 'if' instead.
   Discouraged word: substantial (1 instances); suggest 'large, much' instead.
   Discouraged word: transmit (2 instances); suggest 'send' instead.
Page: rippled v1.3.x Migration Instructions
   Discouraged word: will (1 instances); suggest '(avoid future tense)' instead.
Page: Run rippled as a Validator
   Discouraged word: timely (2 instances); suggest 'prompt' instead.
   Discouraged word: clearly (1 instances); suggest '(omit)' instead.
   Discouraged phrase: similar to (1 instances); suggest 'like' instead.
Page: Run rippled as a Wallet Server
   Discouraged word: wallet (6 instances); suggest 'address, client application, account' instead.
Page: Configure History Sharding
   Discouraged word: allocate (2 instances); suggest 'divide' instead.
Page: Configure Full History
   Discouraged word: sufficient (2 instances); suggest 'enough' instead.
Page: Diagnosing Problems with rippled
   Discouraged word: remainder (1 instances); suggest 'rest' instead.
   Discouraged word: portion (1 instances); suggest 'part' instead.
Page: Understanding Log Messages
   Discouraged word: sufficient (1 instances); suggest 'enough' instead.
   Discouraged phrase: failed to (1 instances); suggest 'didn't' instead.
Page: rippled Server Doesn't Sync
   Discouraged phrase: failed to (1 instances); suggest 'didn't' instead.
   Discouraged word: impacted (1 instances); suggest 'affected, changed' instead.
   Discouraged word: just (1 instances); suggest '(omit)' instead.
Page: Fix SQLite Transaction Database Page Size Issue
   Discouraged word: encounter (3 instances); suggest 'meet' instead.
   Discouraged word: easily (1 instances); suggest '(omit)' instead.
Page: Currency Formats
   Discouraged word: encounter (1 instances); suggest 'meet' instead.
Page: Request Formatting
   Discouraged word: will (2 instances); suggest '(avoid future tense)' instead.
Page: Response Formatting
   Discouraged word: will (3 instances); suggest '(avoid future tense)' instead.
Page: Serialization Format
   Discouraged phrase: in addition (1 instances); suggest 'also, besides, too' instead.
   Discouraged phrase: similar to (1 instances); suggest 'like' instead.
Page: Public rippled Methods
   Discouraged phrase: is authorized to (1 instances); suggest 'may' instead.
Page: ledger
   Discouraged word: regarding (1 instances); suggest 'about, of, on' instead.
Page: tx
   Discouraged word: sufficient (1 instances); suggest 'enough' instead.
Page: deposit_authorized
   Discouraged phrase: is authorized to (2 instances); suggest 'may' instead.
   Discouraged word: sufficient (1 instances); suggest 'enough' instead.
Page: subscribe
   Discouraged word: establish (2 instances); suggest 'set up, create, prove' instead.
Page: server_info
   Discouraged word: will (1 instances); suggest '(avoid future tense)' instead.
Page: server_state
   Discouraged word: will (1 instances); suggest '(avoid future tense)' instead.
Page: wallet_propose
   Discouraged word: simple (1 instances); suggest '(omit)' instead.
Page: logrotate
   Discouraged word: will (3 instances); suggest '(avoid future tense)' instead.
Page: manifest
   Discouraged word: portion (1 instances); suggest 'part' instead.
Page: validators
   Discouraged word: will (2 instances); suggest '(avoid future tense)' instead.
Page: Check
   Discouraged phrase: similar to (1 instances); suggest 'like' instead.
Page: LedgerHashes
   Discouraged phrase: in the process of (1 instances); suggest '(omit)' instead.
Page: Transaction Common Fields
   Discouraged word: performed (1 instances); suggest 'done' instead.
Page: CheckCash
   Discouraged phrase: similar to (1 instances); suggest 'like' instead.
Page: rippled Commandline Usage Reference
   Discouraged word: therefore (1 instances); suggest 'so' instead.
   Discouraged phrase: in addition (1 instances); suggest 'also, besides, too' instead.
Page: Peer Crawler
   Discouraged phrase: similar to (1 instances); suggest 'like' instead.
Page: Validator List Method
   Discouraged phrase: similar to (1 instances); suggest 'like' instead.
Page: XRP-API
   Discouraged word: will (5 instances); suggest '(avoid future tense)' instead.
   Discouraged word: assist (1 instances); suggest 'aid, help' instead.
   Discouraged word: simple (1 instances); suggest '(omit)' instead.
Page: All XRP-API Methods
   Discouraged phrase: and/or (1 instances); suggest '__ or __ or both' instead.
Page: Get Account Transactions
   Discouraged word: selection (1 instances); suggest 'choice' instead.
Page: Transact Methods
   Discouraged phrase: and/or (1 instances); suggest '__ or __ or both' instead.
Page: Prepare a Payment
   Discouraged word: will (3 instances); suggest '(avoid future tense)' instead.
   Discouraged phrase: and/or (1 instances); suggest '__ or __ or both' instead.
Page: Sign and/or submit a payment
   Discouraged phrase: and/or (3 instances); suggest '__ or __ or both' instead.
Page: Create, sign, and submit a payment
   Discouraged word: will (2 instances); suggest '(avoid future tense)' instead.
Page: XRP-API Data Types
   Discouraged word: will (1 instances); suggest '(avoid future tense)' instead.
Page: RippledServerInfo
   Discouraged word: performed (1 instances); suggest 'done' instead.
   Discouraged word: just (1 instances); suggest '(omit)' instead.
   Discouraged phrase: in order to (1 instances); suggest 'to' instead.
Page: Load
   Discouraged word: performed (1 instances); suggest 'done' instead.
Page: Payment object and instructions
   Discouraged word: will (2 instances); suggest '(avoid future tense)' instead.
Page: Transaction Instructions
   Discouraged word: will (1 instances); suggest '(avoid future tense)' instead.
Page: TransactionStatus
   Discouraged word: will (1 instances); suggest '(avoid future tense)' instead.
Page: RippleAPI Reference
   Discouraged phrase: in order to (3 instances); suggest 'to' instead.
   Discouraged word: sufficient (1 instances); suggest 'enough' instead.
   Discouraged phrase: similar to (3 instances); suggest 'like' instead.
   Discouraged word: retain (1 instances); suggest 'keep' instead.
   Discouraged word: facilitate (1 instances); suggest 'ease, help' instead.
   Discouraged word: will (48 instances); suggest '(avoid future tense)' instead.
   Discouraged word: wallet (7 instances); suggest 'address, client application, account' instead.
   Discouraged word: proceed (1 instances); suggest 'do, go ahead, try' instead.
   Discouraged phrase: hot wallet (2 instances); suggest 'operational address' instead.
   Discouraged phrase: in order for (2 instances); suggest 'for' instead.
   Discouraged phrase: a number of (1 instances); suggest 'some' instead.
   Discouraged word: identical (2 instances); suggest 'same' instead.
   Discouraged word: performed (2 instances); suggest 'done' instead.
   Discouraged word: just (3 instances); suggest '(omit)' instead.
   Discouraged phrase: is responsible for (1 instances); suggest '(omit) handles' instead.
   Discouraged phrase: and/or (2 instances); suggest '__ or __ or both' instead.
   Discouraged word: regarding (3 instances); suggest 'about, of, on' instead.
   Discouraged word: subsequently (17 instances); suggest 'after, later, then' instead.
   Discouraged phrase: in addition (1 instances); suggest 'also, besides, too' instead.
   Discouraged word: subsequent (1 instances); suggest 'later, next' instead.
Page: Ripple Data API v2
   Discouraged phrase: time period (11 instances); suggest '(either one)' instead.
   Discouraged phrase: during the period (1 instances); suggest 'during' instead.
   Discouraged word: will (1 instances); suggest '(avoid future tense)' instead.
   Discouraged word: designate (1 instances); suggest 'appoint, choose, name' instead.
Page: xrp-ledger.toml File
   Discouraged phrase: similar to (1 instances); suggest 'like' instead.
   Discouraged word: will (1 instances); suggest '(avoid future tense)' instead.
   Discouraged word: consult (1 instances); suggest 'see' instead.
   Discouraged word: desire (1 instances); suggest 'want, wish' instead.
Page: Use an Escrow as a Smart Contract
   Discouraged word: performed (1 instances); suggest 'done' instead.
```

I introduced a couple of intentional typos and misspellings but the checker didn't seem to catch those:

```
page: About Canceling a Transaction
   Discouraged phrase: a number of (1 instances); suggest 'some' instead.
```

Which doesn't catch the two errors I added in that file:

```
# About Canceling a Transaction

An impotant and intentional feature of the XRP Ledger is that a [transaction](transaction-basics.html)'s outcome is [final](finality-of-results.html) as soon as it has been incorporated in a [ledger version](ledgers.html) that is validated by the [consensus process](consensus.html).

Argke

If a transaction has _not_ yet [....]
```

File: content/_img-sources/cryptographic-keys.uxf [hunk 1]
```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<diagram program="umlet" version="14.2">
```
Comment (mDuo13) [inline content/_img-sources/cryptographic-keys.uxf:2]: This is the source file for `cryptographic-keys.svg`. I used [UMLet](https://umlet.com/) to make it. Reviewers don't need to review this file directly.

File: content/concepts/payment-system-basics/accounts/cryptographic-keys.md [hunk 2]
```diff
@@ -110,18 +125,17 @@ The XRP Ledger supports the following cryptographic signing algorithms:
...
+| `ed25519`   | [EdDSA](https://tools.ietf.org/html/rfc8032) using the elliptic curve [Ed25519](https://ed25519.cr.yp.to/) | This is a newer algorithm which has better performance and other convenient properties. Since Ed25519 public keys are one byte shorter than secp256k1 keys, `rippled` prefixes Ed25519 public keys with the byte `0xED` so both types of public key are 33 bytes. |
 
 When you generate a key pair with the [wallet_propose method][], you can specify the `key_type` to choose which cryptographic signing algorithm to use to derive the keys. If you generated a key type other than the default, you must also specify the `key_type` when signing transactions.
 
 The supported types of key pairs can be used interchangeably throughout the XRP Ledger as master key pairs, regular key pairs, and members of signer lists. The process of [deriving an address](accounts.html#address-encoding) is the same for secp256k1 and Ed25519 key pairs.
 
-**Note:** Currently, you cannot sign [payment channel claims](use-payment-channels.html) with Ed25519 keys. This is a bug.
```
Comment (mDuo13) [inline content/concepts/payment-system-basics/accounts/cryptographic-keys.md:118]: fixed in https://github.com/ripple/rippled/pull/3116 (rippled v1.4.0)

**Comment by mDuo13** [comment]
This diff is already unmanageably big, so I'm going to leave it here for now, with future changes to come in a later PR.

To be clear, the style checker does not give this a pass. There are 354 discouraged words/phrases still present and 164 pages that don't meet their "readability" goals. 

This _does_ get the spell checker to pass. (Also, some context: you need to upgrade to Dactyl v0.12.0 to run the spell checker, which is why Ryan's tests earlier didn't catch the typos.)

**Pull request merged**